### ISSUE TITLE
Create a map using LeaftLet

### ISSUE THREAD

**Issue by turicas** [okfn-brasil/violencia-nao-tem-partido#5]
We have city and state, so we can geolocate these points and then use LeaftLet to show a simple map. [Example here](http://github.com/turicas/feiras-organicas-curitiba/).

**Comment by ellisonleao** [comment]
i can try this one

**Comment by vgeorge** [comment]
If input data has a column identifying the city with an `IBGE id` it will be possible to use coordinates from [this file](https://github.com/mapaslivres/localidades/blob/master/data/cities.csv).

**Comment by vgeorge** [comment]
A "cities" file could added to the Google Sheets file as an auxiliary tab. At "Casos" tab the coordinates could be fetched using VLOOKUP searching for `ibge_city_id`, so there would not exist the need to georeference at the front-end.

**Comment by ellisonleao** [comment]
@turicas are the lat, lng codes going to be populated in the spreadsheet ?

**Comment by turicas** [comment]
@ellisonleao yes! In the "case" spreadsheet. :) I can populate it, we need to change the models to load it and then use it on the frontend.

**Comment by ellisonleao** [comment]
@turicas ok, can you add a fixture sample as well so i can test on my side, please?

**Comment by vgeorge** [comment]
I made a copy of the table and added a mechanism to generate coordinates of the cases from city codes:

https://docs.google.com/spreadsheets/d/1UlkevRSDmnqMDxaE_mjNqxf_y-paGd6qbYAIpgVBL-g/edit?usp=sharing

**Comment by ellisonleao** [comment]
hey @vgeorge, thanks for the help. Looks like the lat and lng fields are going to be a in a single column, splitted by `,`. E.g: `12.23,-5.234` . The current code that i am using is parsing in that way.

**Comment by ocefpaf** [comment]
The approach in #14 is probably better but here is an example on how to achieve this with `geopy` and `folium`. The `folium` map can be saved as a stand-alone HTML.

http://nbviewer.jupyter.org/gist/ocefpaf/3bb0a42943aad9560642e0088df936f8

**Comment by vgeorge** [comment]
@ellisonleao I've changed the file, now the coordinates are stored in a `lonlat` field.

**Comment by vgeorge** [comment]
@ellisonleao I don't have write access to the official doc, someone has to add the changes there.
